renovate.json

```diff
@@ -2,5 +2,14 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "local>ExpediaGroup/renovate-config"
+  ],
+  "includePaths": [
```

Comment:
Is there `excludePaths` option? so we can point to the `code` module only and keep it active on all other modules by default.

Reply:
Yes, but these paths were not detected by renovate in the first place, so I wanted to make sure they are included.